tests/benchmark/read_gbq_colab/aggregate_output.py

```diff
@@ -56,9 +54,8 @@ def aggregate_output(
         project_id,
         dataset_id,
         table_id,
-        session,
         suffix,
-    ) = utils.get_configuration(include_table_id=True)
+    ) = utils.get_configuration(include_table_id=True, include_session=False)
```

Comment:
This parameter doesn't actually exist. Might be nice to have a `start_session=False` argument, though.